opencensus/metrics_quickstart/main.go

```diff
@@ -65,10 +65,8 @@ func main() {
 	// Flush must be called before main() exits to ensure metrics are recorded.
 	defer exporter.Flush()
 
-	view.RegisterExporter(exporter)
-
-	// The minimum reporting period for Stackdriver is 1 minute.
-	view.SetReportingPeriod(60 * time.Second)
+	exporter.StartMetricsExporter()
```

Comment:
Should we check if this returns an error?

Reply:
added the check.